packages/mdc-tab/mdc-tab.scss

```diff
@@ -129,13 +127,12 @@
 }
 
 .mdc-tab--active {
-  @include mdc-tab-text-label-color(primary);
```

Comment:
This breaks CSS Vars for label and icon color. We might need to revert this change.

Reply:
There's a PR to fix this I think: https://github.com/material-components/material-components-web/pull/4688